src/components/molecules/plan/plan.scss

```
@@ -0,0 +1,63 @@
+@use "../../variables" as *;
+
+.plan {
+  text-align: center;
+  width: 100%;
+
+  &:nth-child(odd) {
+    .plan-title {
+      background-color: $orange_zorba;
+    }
+  }
+  &:nth-child(even) {
+    box-sizing: border-box;
+    border: solid 5px $orange_bistre;
+    .plan-title {
+      background-color: $orange_bistre;
+    }
+  }
+
+  @media (max-width: $breakpoint-sm) {
+    margin-bottom: 10px;

```

画面幅ｓの時に margin-bottomがついてほしいのでおそらくコードが以下のようになると思います
```suggestion
  @media (max-width: $breakpoint-md) {
    margin-bottom: 24px;
```